### PR TITLE
Navigate to user's profile on Flow directly when clicked

### PR DESCRIPTION
**Pull request by divad12**
... instead of showing a pop-up liked we used to:

image

Clicking a user will just go directly to the user's profile in the app (the "View Sandy on Flow" option). You can view the user on Facebook from their profile page on the action bar:

image

It might be nice to have the "view on Facebook" button directly on the action bar (instead of hidden in additional options), but need to find a suitable FB icon for that.

### CODE REVIEW

**Comment by jasperfung** [comment]
Agree ^^^
I'll make a FB icon for the actionbar when I jump back onto Flow work

File: res/menu/profile_menu.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+    <!-- TODO(david): Find an icon for this and show on bar if room -->
+    <item android:id="@+id/action_view_on_fb"
+          android:title="@string/action_view_on_fb"/>
+</menu>
```
Comment (jasperfung) [inline res/menu/profile_menu.xml]: nit: newline

Comment (divad12) [inline res/menu/profile_menu.xml]: Done. Vim usually adds this automatically for me. :)

File: src/com/uwflow/flow_android/adapters/CourseReviewAdapter.java [hunk 2]
```diff
@@ -31,7 +30,7 @@
     public CourseReviewAdapter(List<Review> reviews, Context context, FragmentManager fragmentManager) {
         mReviews = reviews;
         mContext = context;
-	mFragmentManager = fragmentManager;
+	    mFragmentManager = fragmentManager;
```
Comment (divad12) [inline src/com/uwflow/flow_android/adapters/CourseReviewAdapter.java]: Oops, fixed the indentation here.

**Comment by jasperfung** [comment]
LGTM

File: src/com/uwflow/flow_android/fragment/ProfileFragment.java [hunk 3]
```diff
@@ -227,32 +262,34 @@ public void getUserCoursesCallback(UserCourseDetail userCourseDetail) {
     }
 
     protected void populateData() {
-        if (user != null) {
```
Comment (divad12) [inline src/com/uwflow/flow_android/fragment/ProfileFragment.java:230]: There's no changes in this function except reducing the nesting via an early return as you can see here:

https://github.com/UWFlow/flow-android/pull/23/files?w=1

(Viewing diff w/o whitespace changes.)

Sorry this got into this PR.

**Comment by divad12** [comment]
@jasperfung Maybe we could use http://android-ui-utils.googlecode.com/hg/asset-studio/dist/icons-actionbar.html to generate an FB icon for the action bar?

**Pull request merged**